papermill/cli.py

```diff
@@ -50,6 +58,8 @@
     '--start_timeout', type=int, default=60, help="Time in seconds to wait for kernel to start."
 )
 @click.option('--report-mode/--not-report-mode', default=False, help="Flag for hiding input.")
+@click.option('--version', default=1.0, is_flag=True, callback=print_the_paermill_version,
```

Comment:
You don't want the default section here.

Reply:
done!